package.json

```diff
@@ -63,7 +63,7 @@
     "@storybook/react": "^3.3.14",
     "babel-cli": "^6.24.1",
     "babel-core": "^6.25.0",
-    "babel-eslint": "^7.1.1",
+    "babel-eslint": "^8.2.3",
```

Comment:
eslint started to give me errors. not sure why, but i upgraded & they went away. upgrading fixed a few bugs so it caught a few more errors across the app.